### ISSUE TITLE
Import directive into component

### ISSUE THREAD

**Issue by tomekole** [nil/v-drag#41]
Is it possible to import directive to individual component in the way shown below?

`import ??? from "v-drag";`

`export default defineComponent({`
`  directives:{???},`
`})`

The use case are nested components distributes as Custom Elements (Web Components), where app.use() is not working. All imports must be done in the component itself.

https://vuejs.org/guide/extras/web-components.html#building-custom-elements-with-vue

Thanks!

**Comment by blazefireio** [comment]
I dont think so. You need to code this yourself and find a way how it could work.

**Comment by mukhrr** [comment]
@tomekole you can create any global js file and there 
```
import Vue from 'vue';
import drag from "v-drag"

Vue.use(drag, {
  removeTransition: false
})
```
